tool/db/types.go

```diff
@@ -20,8 +20,14 @@ const (
 
 // L2Block is a full l2 block
 type L2Block struct {
-	L2BlockStart
-	L2BlockEnd
+	BatchNumber    uint64         // 8 bytes
+	L2BlockNumber  uint64         // 8 bytes
+	Timestamp      int64          // 8 bytes
+	GlobalExitRoot common.Hash    // 32 bytes
+	Coinbase       common.Address // 20 bytes
+	ForkID         uint16         // 2 bytes
+	BlockHash      common.Hash    // 32 bytes
```

Comment:
BlockHash and StateRoot are needed? it seems we are not assigning any value to it, also reviewing the specs this fields doesn't exists in the L2Block entry